src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,10 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        Machine[] machines =  {new Truck(), new Bulldozer(), new Excavator()};
+        for (int i = 0; i < machines.length; i++) {
+            machines[i].doWork();
+        }
```

Comment:
Why don`t use stopWork?

Reply:
Excuse me, did yopu check "solution1" branch? I dont understand where is mistakes